src/main/java/Ex3.java

```diff
@@ -0,0 +1,19 @@
+import java.util.ArrayList;
+import java.util.List;
+public class Ex3 {
+    public static String PTTSNT(int n){
+        String s = "";
```

Comment:
Tránh khai báo tên tối nghĩa, ở đây có thể là 
```suggestion
        String result = "";
```